packages/prbnmcn-stats/prbnmcn-stats.0.0.3/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Basic statistics"
```

Comment:
Would you be able to push a description (even if short) to this opam file?

Reply:
Sure, it's done